.github/workflows/managed.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        use-cache: [true]
```

Comment:
by default don't use cache, as of know it is faster and CI doesn't fail due to low disk space